projects/cdk/pipes/replace/replace.pipe.ts

```diff
@@ -0,0 +1,30 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+type SearchValue = RegExp | string;
+
+type ReplaceValue = string | ((substring: string, ...args: any[]) => string);
+
+@Pipe({name: `tuiReplace`})
+export class TuiReplacePipe implements PipeTransform {
+    transform(
+        value: null | undefined,
+        searchValue: SearchValue,
+        replaceValue: ReplaceValue,
+    ): undefined;
```

Comment:
I don't think it makes sense to overload, maybe just `return value?.replace(...) || ''`?

Reply:
fixed